File: front-end/src/models/CheckListDay.js
export class CheckListDay {
    DayId;
    ShopeStoreId;
    CheckLists;
    Day;
    Rank;


    constructor(DayId,ShopeStoreId, CheckLists, Day, Rank ) {
        this.DayId = DayId;
        this.ShopeStoreId=ShopeStoreId
        this.CheckLists = CheckLists;
        this.Day = Day;
        this.Rank = Rank;
    }
}

File: back-end/StoreOperation.WebApi/Data/Entities/CheckListDay.cs
using System;
using System.Collections.Generic;

namespace StoreOperation.WebApi.Data.Entities
{
    public partial class CheckListDay
    {
        public CheckListDay()
        {
            CheckLists = new HashSet<CheckList>();
        }
        public Guid CheckListDayId { get; set; }
        public DateTime Day { get; set; }
        public Guid StoreId { get; set; }
        public AppStore Store { get; set; }
        public ICollection<CheckList> CheckLists { get; set; }

        public int Rank { get; set; }
        /// <summary>
        /// aktif/pasif
        /// </summary>
        public bool State { get; set; }
    }
}
